File: ssaExample/types_declInfo.java
import java.util.Map;

public class types_declInfo {

	public types_Scope file;
	public types_Var[] lhs;
	public Object vtyp;
	public Object init;
	public Boolean inherited;
	public ast_TypeSpec tdecl;
	public ast_FuncDecl fdecl;
	public Map<Object, Boolean> deps;
}

File: ssaExample/types_declInfo.kt
class types_declInfo {

	var file: types_Scope? = null
	var lhs: Array<types_Var>? = null
	var vtyp: Any? = null
	var init: Any? = null
	var inherited: Boolean? = null
	var tdecl: ast_TypeSpec? = null
	var fdecl: ast_FuncDecl? = null
	var deps: Map<Any, Boolean>? = null
}
